apps/web/src/app/admin-console/common/base-members.component.ts

```diff
@@ -222,6 +222,11 @@ export abstract class BaseMembersComponent<UserView extends UserViewTypes> {
       }
       await confirmUser(publicKey);
     } catch (e) {
+      this.toastService.showToast({
+        variant: "error",
+        title: this.i18nService.t("errorAdminConfirmUser"),
```

Comment:
instead of using the translation (which would fail and fall back to the default language) you could probably just use the provided string directly.